builtin/game/item_entity.lua

```diff
@@ -59,17 +59,20 @@ core.register_entity(":__builtin:item", {
 			math.floor(def.light_source / 2 + 0.5)
 
 		local size_bias = 1e-3 * math.random() -- small random bias to counter Z-fighting
+		local c = {-size, -size, -size, size, size, size}
```

Comment:
What's the point of this? Unless you move `local c` out of the function (which you can't because it depends on `size`), it will still be creating the table for every object (I do now wonder whether the collisionbox should be affected by the size bias BTW).

Reply:
The issue is with the `get_properties()` call because of
* general C++ call overhead
* it pushes a brand new table every time

Reply:
> The issue is with the `get_properties()` call because of
> 
>     * general C++ call overhead
> 
>     * it pushes a brand new table every time

Yeah I realized this after I had posted the comment, hence I edited it.